microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -3,102 +3,212 @@ description: The latest experimental features in Microsoft Edge DevTools
+| Off | [Timeline: Replay input events](#timeline-replay-input-events) |
+| Off | [Enable dynamic Welcome content](#enable-dynamic-welcome-content) |
+
+<!--| Off | [CSS Overview](../whats-new/2020/10/devtools#view-and-fix-color-contrast-issues-in-the-css-overview-tool) |-->
+<!--| Off | Empty sourcemap auto-stepping |-->
+<!--| Off | Source diff |-->
 
 <!-- ====================================================================== -->
 ## Turning on experimental features
 
 To turn on (or off) experimental features in Microsoft Edge:
 
-1.  [Open DevTools](../open/index.md).  To do this, in Microsoft Edge, select the **Settings and more** button, which is an ellipses (three dots).  Then hover over **More tools**, and then select **Developer tools**.
+1.  [Open DevTools](../open/index.md). In Microsoft Edge, click the **Settings and more** button, which is an ellipsis (three dots). Then hover over **More tools**, and click **Developer tools**.
 
-1.  Open the [Settings](../customize/index.md#settings) pane of DevTools.  To do this, select the **Settings** (gear) icon.
+1.  Click the **Settings** (gear) icon to open the [Settings](../customize/index.md#settings) pane of DevTools.
 
-1.  On the left side of the **Settings** pane, select the **Experiments** section.
+1.  On the left side of the **Settings** pane, click the **Experiments** section.
 
     :::image type="content" source="../media/experiments-devtools.msft.png" alt-text="The Experiments page in Settings" lightbox="../media/experiments-devtools.msft.png":::
 
-1.  On the **Experiments** page, scroll through the list of all available experimental features and select the checkbox next to each feature that you want to test.  Some experiments are turned on by default.
+1.  On the **Experiments** page, scroll through the list of all available experimental features. Click an empty checkbox to select each feature that you want to test. Some experiments are turned on by default.
+
+1.  Click a selected checkbox to turn off any feature you do not want to test.
 
-1.  Select the **X** in the upper right to close **Settings**.
+1.  Click the **X** in the upper right to close **Settings**.
 
-1.  Select the **Reload DevTools** button.
+1.  Click the **Reload DevTools** button.
 
 > [!NOTE]
-> Experimental features are constantly being updated and might cause performance issues.  To turn off an experimental feature, open the **Experiments** page and clear the checkbox of the experimental feature that you want to turn off.
+> Experimental features are constantly being updated and might cause performance issues. To turn off an experimental feature, open the **Experiments** page and clear the checkbox of the experimental feature that you want to turn off.
 
 
 <!-- ordering for the h2 sections below: same order as in the Experiments page.  if change to a different ordering scheme, note that scheme here. -->
 
 
```

Comment:
\#\# List of experiments
The following experiments could be unstable or unreliable and may require you to restart DevTools.

ie mirror the UI strings here, that group the checkboxes into 2 groups.

h2 strategy: flattened; since the "In this article" feature shows h2's only, add the two "warning" strings (or equivalent headers) as h2 headings, but keep their children's headings as h2's as well.

Reply:
Added note style "warning"